words/compile.asm

```diff
@@ -159,8 +159,14 @@ cmpl_as_call:
         ; Stack is either ( xt xt nt ) or ( xt xt' u )
         ; Use the xt or xt' (in the middle) as the jsr address
         ; so that strip-underflow is respected.
+                lda tmp3                ;TODO hack check if we had stack juggling
```

Comment:
this is fixed slightly differently in the "remove UF flag" branch but the idea is the same, since the optimization is unsafe if the word also had a stack juggling prologue